pkg/server/server.go

```diff
@@ -676,10 +678,21 @@ func makeFilesResult(files map[string]streamer.File) *pb.FilesResult {
 	return &res
 }
 
-func BuildCredsFromSSHConfig(login, password, host, sshConfigPassphrase string, logger *zap.Logger) (credentials.Credentials, error) {
-	privateKeys, err := credentials.GetPrivateKeysFromConfig(host)
-	if err != nil {
-		return nil, err
+func BuildCredsFromSSHConfig(login, password, host, sshConfigPassphrase, privateKeyPath string, logger *zap.Logger) (credentials.Credentials, error) {
+	var privateKeys [][]byte
+	if len(privateKeyPath) > 0 {
+		key, err := os.ReadFile(privateKeyPath)
+		if err != nil {
+			log.Fatalf("Unable to read private key: %v", err)
```

Comment:
I think this log is superfluous. The caller will log it anyway, so it is a duplicate. Also it will terminate the program:
`Fatalf is equivalent to [Printf](https://pkg.go.dev/log#Printf) followed by a call to [os.Exit](https://pkg.go.dev/os#Exit)(1).`